src/main.ts

```diff
@@ -15,23 +15,37 @@ export class Main extends Command {
   }
 
   async run() {
+    if (this.argv.length === 0) return this._runDefaultCommand()
     let [id, ...argv] = this.argv
     this.parse({strict: false, '--': false, ...this.ctor as any})
     if (!this.config.findCommand(id)) {
       let topic = this.config.findTopic(id)
       if (topic) return this._help()
+      return this._runDefaultCommand()
     }
     await this.config.runCommand(id, argv)
   }
 
+  protected _getDefaultCommand(): string | undefined {
+    const oclif = this.config.pjson.oclif as any
+    return oclif.defaultCommand
+  }
+
+  protected async _runDefaultCommand() {
+    const defaultCommandId = this._getDefaultCommand()
+    if (defaultCommandId) {
+      await this.config.runCommand(defaultCommandId, [...this.argv])
+      return
+    } else return this._help()
+  }
+
   protected _helpOverride(): boolean {
-    if (['-v', '--version', 'version'].includes(this.argv[0])) return this._version() as any
-    if (['-h', 'help'].includes(this.argv[0])) return true
-    if (this.argv.length === 0) return true
-    for (let arg of this.argv) {
-      if (arg === '--help') return true
-      if (arg === '--') return false
+    if (this.argv.length === 0) {
```

Comment:
Likewise, remove this bit.

Reply:
When no arguments are passed to the CLI, I want the default command to run instead of showing the help page.